socs/agents/wiregrid_actuator/agent.py

```diff
@@ -242,6 +242,17 @@ def _insert_eject(
             self.log.error(msg)
             return False, msg
 
+        # Additional small moving to ensure the limit switch ON
+        # No limit switch check during this moving
+        # Only this function reflects pos/neg of <distance>.
+        # Please use +/- distance for insertion/ejection, respectively.
+        ret = self.actuator.move(0.5 * (float)(1. if is_insert else -1.), speedrate=0.2)
+        if not ret:
+            msg = '_insert_eject(): ERROR!:'\
+                  'Failed to move in the additional small moving after the last slow moving.'
```

Comment:
```suggestion
                  'Failed to move in the additional small move after the last slow move.'
```

Just a small grammar suggestion.